Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -633,7 +633,7 @@ class BrowserViewController: UIViewController,
 
     private func prepareURLOnboardingContextualHint() {
         guard contextHintVC.shouldPresentHint(),
-              featureFlags.isFeatureEnabled(.contextualHintForToolbar, checking: .buildOnly)
+              featureFlags.isFeatureEnabled(.isToolbarCFREnabled, checking: .buildOnly)
```

Comment:
Slight concern here, as this is for contextual hint for toolbar but then we have an additional flag, wondering if we need to check both here `checkNimbusForContextualHintsFeature` ?